testdata/channel.txt

```diff
@@ -12,9 +12,7 @@ received num: 8
 recv from closed channel: 0 false
 complex128: (+7.000000e+000+1.050000e+001i)
 sum of n: 149
-sum: 25
-sum: 29
-sum: 33
+sum of sums: 87
```

Comment:
These numbers were spread out differently when using threading. Sometimes the right numbers came out, but other times the numbers were balanced differently. Of course that's all correct: the important part is that every number is received by exactly one receiver.

Reply:
The mathematician in me really wants this to switch from adding integers to multiplying unique primes so that we can be sure (due to https://en.wikipedia.org/wiki/Fundamental_theorem_of_arithmetic ) that everything was received exactly once.